js/v6decode.js

```diff
@@ -267,12 +269,12 @@ function updateTeredo(address) {
 }
 
 function updateArinJson(address) {
-  $.getJSON('/arin/ip/' + address.correctForm(), function getArinJson(data) {
+  $.getJSON('https://whois.arin.net/rest/ip/' + address.correctForm(), function getArinJson(data) {
```

Comment:
does ARIN support CORS now? (this was proxied through apache so I could add CORS headers)

Reply:
Yes, you can find a running version on my v-server [here](https://v6decode.v22018086706171227.nicesrv.de)